R/generate_template.R

```diff
@@ -12,6 +12,7 @@
 #' @param name name for the new application and directory
 #' @param location base path for creation of \code{name}
 #' @param sampleapp whether to create a sample shiny application
+#' @param right_sidebar parameter to set the right_sidebar. It can be TRUE/FALSE or a custom icon. See ?shiny::icon() for how to create an icon.
 #'
```

Comment:
Add an **@section Right Sidebar** that has note about the right sidebar icon usage and parameters.  IE False = no sidebar, True = sidebar with XYZ icon, etc.

Reply:
looks good - mention what the default icon is equivalent to please. (aka TRUE    --- sidebar with default icon ('gears') ?  I don't know what the default icon's name would be, so use that of course.